kubernetes/third_party/swagger-ui/index.md

```diff
@@ -0,0 +1,4 @@
+---
+
+Kubernetes swagger UI has now been replaced by our generated API reference docs
+which can be accessed at http://kubernetes.io/docs/api-reference/README/.
```

Comment:
Sent https://github.com/kubernetes/kubernetes.github.io/pull/1928 to add missing ones